src/routes/locking/locking.service.ts

```diff
@@ -68,6 +69,31 @@ export class LockingService {
     };
   }
 
+  async getLeaderBoardByCampaignId(args: {
```

Comment:
As before regarding "leaderboard":
```suggestion
  async getLeaderboardByCampaignId(args: {
```

Reply:
I'm resolving this one as this got renamed to `getCampaignLeaderboard`. Lmk if you have other suggestion of course!